src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -708,4 +708,10 @@ The default is 'true' if a runtime identifier is specified.</value>
   <data name="CorruptSolutionProjectFolderStructure" xml:space="preserve">
     <value>The solution file '{0}' is missing EndProject tags or has invalid child-parent project folder mappings around project GUID: '{1}'. Manually repair the solution or try to open and save it in Visual Studio."</value>
   </data>
+  <data name="ArtifactsPathOptionDescription" xml:space="preserve">
```

Comment:
It does include intermediate files.  I didn't include that in the description because I don't expect people to be looking for the intermediate path in order to do something with the files there.

Note that if you set the `ArtifactsPath` property in the project file instead of in Directory.Build.props or on the command line, that it doesn't affect the intermediate output path.